example/java/build_library.sh

```diff
@@ -22,7 +22,7 @@ gradle javaDocJar
 cp build/shared_model/bindings/libs/* dist/
 
 cd build/shared_model/bindings
-jar -cvf iroha_lib.jar *.java
+jar -cvf iroha_lib.jar jp/co/soramitsu/iroha/*.java
```

Comment:
Can be `**/*.java`, though not sure that is working for all of the shells

Reply:
Well, let's not try :)